src/iop/channelmixerrgb.c

```diff
@@ -3868,6 +4177,85 @@ void gui_init(struct dt_iop_module_t *self)
 
   gtk_box_pack_start(GTK_BOX(self->widget), GTK_WIDGET(hbox), FALSE, FALSE, 0);
 
+  g->spot_settings = dt_bauhaus_combobox_new(self);
+  dt_bauhaus_widget_set_label(g->spot_settings, NULL, N_("spot color mapping settings"));
```

Comment:
Just a minor thing from a brief glance at the screenshots but I think the word "settings" is redundant here, since all of the controls in a module are "settings".